model/enduser/registry.yaml

```
@@ -0,0 +1,18 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes information about the end user, which can be used as a subdomain of browser, client, or user domains.
+    attributes:
+      - id: enduser.pseudo.id
+        type: string
+        stability: experimental
```

we use development now, let's change it here and in other non-deprecated attributes

```suggestion
        stability: development 
```